test/ctia/stores/es/init_test.clj

```diff
@@ -87,6 +87,52 @@
       (is (= "2s" refresh_interval)))
     (index/delete! conn (str indexname "*"))))
 
+(deftest get-existing-indices-test
+  (index/delete! es-conn (str indexname "*"))
+  (let [exited? (atom false)
+        fake-exit (fn [] (reset! exited? true))]
+    (with-redefs [sut/system-exit-error fake-exit]
+      (let [test-fn (fn [msg
+                         input-indexname
+                         expected-exited?
+                         expected-output]
+                      (testing msg
+                        (let [ouput (sut/get-existing-indices es-conn input-indexname)]
+                        (when-not expected-exited?
+                          (is (= expected-output ouput)))
+                        (is (= expected-exited? @exited?))
+                        (reset! exited? false))))
```

Comment:
You can move the `(reset! exited? false)` outside and above the `testing`, so that it's clearer that there are two phases ('setup' + 'exercise')